pkg/dataplane/http/context.go

```diff
@@ -1719,11 +1719,14 @@ func (c *context) PutOOSObjectSync(putOOSObjectInput *v3io.PutOOSObjectInput) er
 		buffer.Write(ioVec)
 	}
 
-	headers := putOOSObjectHeaders
```

Comment:
can delete putOOSObjectHeaders?

Reply:
I will break the general flow : all the headers are defined in headers.go